README.md

```diff
@@ -0,0 +1,204 @@
+
+> How to clone and use this project project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- VSCode or any other equivalent code editor
+
+- Ruby and IRB
+
+### Linux
+```sh
+  sudo apt install ruby
+```
+### windows 
+
+[Download Ruby and IRB](https://ruby-lang.org/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command 
+```sh
+git clone https://github.com/tsohleDev/OOP-School-Library.git
+```
+
+- Go to the repository folder using command prompt
+```sh
+ cd OOP-School-Library
+```
+
+### Install
+
+Install this project with:
+
+```sh
+ bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  irb 
+```
+
+<!-- ### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  npm test
+```
+- -->
+
+<!-- ### Deployment
+
+You can deploy this project using:
+
+- To start run 
+```sh
+  npm run build
+``` -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> All of the collaborators of this project.
+
+👤 **Tsohle Mokhemisi**
+
+- GitHub: [@githubhandle](https://github.com/tsohleDev2)
+- Twitter: [@twitterhandle](https://twitter.com/RealTsohle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tsohle-mokhemisi-3687401b2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- **[Create a console app to interact with the app]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> To support this project
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Credits to everyone who inspired the codebase.
+
+I would like to thank [Microverse](https://microverse.org) for the great content
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Please remove this section of the readme as it is part of the template and not required to have a professional README 👍